src/webextension/vpnadapter.h

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef WEBEXTENSION_VPNADAPTER_H
+#define WEBEXTENSION_VPNADAPTER_H
+
+#include <QList>
+
+#include "baseadapter.h"
+
+class ServerCountryModel;
+
+namespace WebExtension {
+
+class VPNAdapter : public BaseAdapter {

```

Comment:
Why no tests for this class?

Reply:
What if you add the tests for these in the general test targets that have these classes mocked out?